shared/js/views.jsx

```diff
@@ -508,6 +509,144 @@ loop.shared.views = (function(_, mozL10n) {
     }
   });
 
+  var CopyLinkButton = React.createClass({
```

Comment:
As we're sharing these only across desktop, and not standalone, I'd prefer for them to go into a `desktopViews.jsx` or something under the add-on directory.

Otherwise, we're adding more styles & code to the standalone which we don't need (and potentially confuses people as to where it is used).